vars/checkPatch.groovy

```diff
@@ -46,7 +46,7 @@ def call(Map config = [:]) {
     // Need the jenkins module to do linting
     checkoutScm url: 'ssh://review.hpdd.intel.com:29418/exascale/jenkins',
                 checkoutDir: 'jenkins',
-                credentialsId: 'bf21c68b-9107-4a38-8077-e929e644996a'
+                credentialsId: 'daos-gerrit-read'
```

Comment:
These have the equivalent credentials in jenkins-3 then?

Reply:
Yes, the credential ID has been installed on Jenkins-3.